charmcraft.yaml

```diff
@@ -3,11 +3,11 @@ type: charm
 platforms:
   ubuntu@20.04:amd64:
   ubuntu@22.04:amd64:
+  ubuntu@24.04:amd64:
 
-# This below is needed for charmhelpers 0.20.24 to build
 parts:
   charm:
-    charm-python-packages:
-      - setuptools < 58
+    source: .
+    plugin: charm
```

Comment:
@jonathanudd do you know why some stuff, like this one, appears here although it's already in `main`?